### PR TITLE
DSG-2790/Remove-profileId-from-the-claim-invitation-endpoints

### PR DESCRIPTION
**Pull request by afcalderonm**
## Types of changes

Patch
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Documentation change (documentation only, will not require redeployment)
- [ ] Refactor or cleanup (functionality unchanged)
Minor
- [ ] New feature (non-breaking change which adds functionality)
Major
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have performed a self-review of my own code
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

- [ ] I have confirmed all acceptance criteria defined by the story have been met

### CODE REVIEW

File: service/src/main/java/io/nuvalence/workmanager/service/controllers/IndividualProfilesApiDelegateImpl.java [hunk 1]
```diff
@@ -448,37 +447,29 @@ public ResponseEntity<PageProfileInvitationResponse> getIndividualProfileInvitat
...
-        }
-
-        if (invitations.stream()
-                        .filter(
-                                invitation ->
-                                        invitation.getClaimed() != null && invitation.getClaimed())
-                        .count()
-                > 0) {
-            throw new ConflictException("You have already claimed an invitation to this profile");
+        if (profileInvitation.getClaimed() != null && profileInvitation.getClaimed()) {
```
Comment (Ortega-Dan) [inline service/src/main/java/io/nuvalence/workmanager/service/controllers/IndividualProfilesApiDelegateImpl.java:471]: Validating the email of the authenticated user is needed.

File: service/src/main/java/io/nuvalence/workmanager/service/controllers/EmployerProfilesApiDelegateImpl.java [hunk 2]
```diff
@@ -437,34 +437,28 @@ public ResponseEntity<PageProfileInvitationResponse> getEmployerProfileInvitatio
...
+        ProfileInvitation profileInvitation = profileInvitationOptional.get();
+
+        UUID profileId = profileInvitation.getProfileId();
+
+        Employer employer =
+                employerService
+                        .getEmployerById(profileId)
+                        .orElseThrow(() -> new NotFoundException(EMPLOYER_PROFILE_NOT_FOUND_MSG));
+
+        if (profileInvitation.getClaimed() != null && profileInvitation.getClaimed()) {
```
Comment (Ortega-Dan) [inline service/src/main/java/io/nuvalence/workmanager/service/controllers/EmployerProfilesApiDelegateImpl.java:461]: Same about email.

File: service/src/main/java/io/nuvalence/workmanager/service/controllers/IndividualProfilesApiDelegateImpl.java (hunk 1 above)
Comment (afcalderonm) [reply service/src/main/java/io/nuvalence/workmanager/service/controllers/IndividualProfilesApiDelegateImpl.java:473]: thanks dan, i just add the validation

File: service/src/main/java/io/nuvalence/workmanager/service/controllers/EmployerProfilesApiDelegateImpl.java (hunk 2 above)
Comment (afcalderonm) [reply service/src/main/java/io/nuvalence/workmanager/service/controllers/EmployerProfilesApiDelegateImpl.java:463]: thanks dan, i just add the validation

**Review by Ortega-Dan — APPROVED** [review]

**Pull request merged**